tools/go-cli/go-whisk-cli/main.go

```diff
@@ -20,17 +20,30 @@ import (
     "fmt"
     "os"
     "reflect"
+    "time"
 
     "../go-whisk/whisk"
     "../go-whisk-cli/commands"
 )
 
+// CLI_BUILD_TIME holds the time of the CLI build.  During production builds,
+// this value will be overwritten via the command:
+//     go build -ldflags "-X main.CLI_BUILD_TIME=nnnnn"   // nnnnn is the new timestamp
+var CLI_BUILD_TIME string = "not set"
+
 var cliDebug = os.Getenv("WSK_CLI_DEBUG")  // Useful for tracing init() code
 
 func init() {
     if len(cliDebug) > 0 {
         whisk.SetDebug(true)
     }
+
+    // Initialize a default CLI build timestamp value only if the value was not
+    // set via the go linker flags.
+    if CLI_BUILD_TIME == "not set" {
+        CLI_BUILD_TIME = time.Now().Format("2006-01-02T15:04:05-07:00")
```

Comment:
I'll just leave that local-build default as "not set", so no worries about the confusing Go date format string.  For more clarity, the format string can be in an arbitrary order, but the actual values for the year, etc fields, must be a specific numeric value.  For example, to format the year as a 4 digit year, `2006` must be used, for a 2 digit year `06` must be used.  Go uses numbers instead of alpha chars like 'yyyy' for many of the date format components.